easybuild/easyconfigs/r/Ruby/Ruby-2.7.1-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,148 @@
+
+builddependencies = [('binutils', '2.32')]
+
+exts_default_options = {
+    'source_urls': ['https://rubygems.org/downloads/'],
+    'source_tmpl': '%(name)s-%(version)s.gem',
+}
+
+exts_list = [
+    ('ffi', '1.12.2', {
+        'checksums': ['048ad01d5369f67075f943c16f1058f10663af2a66eedb87d921316ba1828e82'],
+    }),
+    ('childprocess', '3.0.0', {
+        'checksums': ['4579a87cdc962de252eebf1482a4185fad383ae7dbe29a746ba2be8e261280c5'],
+    }),
+    ('json', '2.2.0', {
+        'checksums': ['9dd1437156773f72c096058ec837faac1b00077121a3fd574e68f895ea3aa96b'],
+    }),
+    ('cabin', '0.9.0', {
+        'checksums': ['91c5394289e993e7037a6c869e3f212f31a5984d2b1811ac934f591c87446b2c'],
+    }),
+    ('backports', '3.17.0', {
+        'checksums': ['bb18a4c7a2a13828d18e348ea81183554adcaac4fc9db0ecd1f3d1dfbd7fdc8f'],
+    }),
+    ('arr-pm', '0.0.10', {
+        'checksums': ['0fa2d425a84eeeaf8f95268568e826723b41aeefa14511d1f0ed1368c30bdd1f'],
+    }),
+    ('clamp', '1.3.1', {
+        'checksums': ['0975e919dc76cfb1a63264fec5c2b97c4bba54cbdec7894ab9e77f0e87436a36'],
+    }),
+    ('multipart-post', '2.1.1', {
+        'checksums': ['d2dd7aa957650e0d99e0513cd388401b069f09528441b87d884609c8e94ffcfd'],
+    }),
+    ('multipart-post', '1.2.0', {
+        'checksums': ['ccf26a61706f128a2a3f3745aba0f79fde7b6fee068e03490eb189c2aaa5e78a'],
+    }),
+    ('faraday', '1.0.1', {
+        'checksums': ['381aee04fcc9effbe5fa7cc703d8f5f20293722f987ded4f958f77514cd29373'],
+    }),
+    ('faraday', '0.8.11', {
+        'checksums': ['b16608080bd0874d44b15db85833a75cc6dfcc7e7ffc5e0603d043c559f9bb94'],
+    }),
+    ('faraday_middleware', '1.0.0', {
+        'checksums': ['19e808539681bbf2e65df30dfbe27bb402bde916a1dceb4c7496dbe8de14334a'],
+    }),
+    ('highline', '2.0.3', {
+        'checksums': ['2ddd5c127d4692721486f91737307236fe005352d12a4202e26c48614f719479'],
+    }),
+    ('net-http-pipeline', '1.0.1', {
+        'checksums': ['6923ce2f28bfde589a9f385e999395eead48ccfe4376d4a85d9a77e8c7f0b22f'],
+    }),
+    ('connection_pool', '2.2.2', {
+        'checksums': ['c8cc9446bcc51034103c1259ad70b91dc9f5297d13460b2c0cce7e5a93e8d451'],
+    }),
+    ('net-http-persistent', '3.1.0', {
+        'checksums': ['eab637fcee31d2f2c119c24b5ae36d73cd8c08ccd9a45567829f24059e9836ae'],
+    }),
+    ('net-http-persistent', '2.9.4', {
+        'checksums': ['24274d207ffe66222ef70c78a052c7ea6e66b4ff21e2e8a99e3335d095822ef9'],
+    }),
+    ('multi_json', '1.14.1', {
+        'checksums': ['d971296c0eacea289d31e4a7ab7ac5eda97262c62bbc8c110de4f5e36425c577'],
+    }),
+    ('public_suffix', '4.0.3', {
+        'checksums': ['87a9b64575e6d04a2e83882a2610470ea47132828c96650610b4c511b4c1d3b0'],
+    }),
+    ('addressable', '2.7.0', {
+        'checksums': ['5e9b62fe1239091ea9b2893cd00ffe1bcbdd9371f4e1d35fac595c98c5856cbb'],
+    }),
+    ('addressable', '2.4.0', {
+        'checksums': ['7abfff765571b0a73549c9a9d2f7e143979cd0c252f7fa4c81e7102a973ef656'],
+    }),
```

Comment:
Duplicate extension

Reply:
the duplicates are here to satisfy the [`gh` gem](https://rubygems.org/gems/gh) which was part of th extension in the previous easyconfig -- I thus kept it. However, while this gem was only recently updated (yesterday), yet kept the dependencies as was before i.e. to old versions of the duplicate gems which has to exist under two versions: latest one and the one required for `gh`. They do not seem keen to upgrade the dependencies (see https://github.com/travis-ci/gh/pull/27) 
One solution could be to drop `gh` to avoid duplicates -- but I assumed it was put here for a reason and didn't want to break past workflow.

Reply:
Thanks for the clarification! I went through the list and made some changes to keep the same gems in this easyconfig and have as higher as possible versions of each package. Let me know if this solution is suitable for you.